.github/workflows/workflow.yml

```diff
@@ -17,13 +17,23 @@ jobs:
         os:
           - macos-latest
           - ubuntu-latest
-
     runs-on: ${{ matrix.os }}
-
     steps:
       - name: asdf_plugin_test
         uses: asdf-vm/actions/plugin-test@v1
         with:
           command: ark version
         env:
           GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We no longer need this from the latest version!

Reply:
yep, working on it